contracts/src/protocol/IPTokenStaking.sol

```diff
@@ -50,31 +51,8 @@ contract IPTokenStaking is IIPTokenStaking, Ownable2StepUpgradeable, ReentrancyG
     /// @notice The fee paid to update a validator (unjail, commission update, etc.)
     uint256 public fee;
 
-    /// @notice Verifies that the syntax of the given public key is a 65 byte uncompressed secp256k1 public key.
-    modifier verifyUncmpPubkey(bytes calldata uncmpPubkey) {
-        if (uncmpPubkey.length != 65) {
-            revert Errors.IPTokenStaking__InvalidPubkeyLength();
-        }
-        if (uncmpPubkey[0] != 0x04) {
-            revert Errors.IPTokenStaking__InvalidPubkeyPrefix();
-        }
-        _;
-    }
-
-    /// @notice Verifies that the given 65 byte uncompressed secp256k1 public key (with 0x04 prefix) is valid and
-    /// matches the expected EVM address.
-    modifier verifyUncmpPubkeyWithExpectedAddress(bytes calldata uncmpPubkey, address expectedAddress) {
-        if (uncmpPubkey.length != 65) {
-            revert Errors.IPTokenStaking__InvalidPubkeyLength();
-        }
-        if (uncmpPubkey[0] != 0x04) {
-            revert Errors.IPTokenStaking__InvalidPubkeyPrefix();
-        }
-        if (_uncmpPubkeyToAddress(uncmpPubkey) != expectedAddress) {
-            revert Errors.IPTokenStaking__InvalidPubkeyDerivedAddress();
-        }
-        _;
-    }
+    /// @notice Staking periods and their corresponding durations
+    mapping(IIPTokenStaking.StakingPeriod period => uint32 duration) public stakingDurations;
```

Comment:
Is this used?

Reply:
Good catch, this got there by rebasing I guess, previous PRs removed it